QUARKUS-2479.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-2479 SmallRye Config SecretKeys support
+
+Jira: https://issues.redhat.com/browse/QUARKUS-SmallRye Config SecretKeys support
+
+This test plan covers support of secret keys in SmallRye config
+
+## Scope of the testing
+Following topics should be covered:
+
+1. Verify, that some secrets can be marked as secret 
+2. Verify, that usage of these secrets leads to an exception
+3. Verify, that these secrets can be used only within doUnlocked section
+
```

Comment:
Coverage should ensure JVM and native mode in baremetal and OpenShift

Reply:
+1 for JVM and native
not so sure about the need for coverage on OpenShift